.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+sudo: false
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
+
+matrix:
+  allow_failures:
+  - dart_task: dartanalyzer
```

Comment:
I don't think it makes sense to allow failures that are being actively worked on. I'd just let the bots be red until they go green.